scripts/load_mentors.js

```diff
@@ -143,10 +160,49 @@ exports.handler = async (event) => {
   for (const mentorsChunk of chunks(indexMentors, ELASTIC_CHUNK_SIZE)) {
     i += 1;
     console.log(`Indexing mentors (chunk ${i})...`);
-    const indexBody = mentorsChunk.flatMap(([id, mentor]) => [
-      { index: { _index: ELASTIC_INDEX, _id: id } },
-      mentor,
-    ]);
+    const indexBody = (
+      await Promise.all(
+        mentorsChunk.map(async ([id, mentor]) => {
+          const newMentorObject = { ...mentor };
+          await Promise.all(
+            [
+              [mentor.full_image_url, `${id}-full`, "full_image_url"],
+              [
+                mentor.thumbnail_image_url,
+                `${id}-thumbnail`,
+                "thumbnail_image_url",
+              ],
+            ].map(async ([imageURL, imageKey, objectProperty]) => {
+              if (imageURL === PLACEHOLDER_THUMBNAIL_URL) {
+                s3.deleteObject(
+                  {
+                    Bucket: AWS_S3_BUCKET_NAME,
+                    Key: `${AWS_S3_IMAGE_FOLDER}${imageKey}.jpg`,
+                  },
+                  function (err, data) {}
+                );
+                return [{ index: { _index: ELASTIC_INDEX, _id: id } }, mentor];
+              }
+              const res = await fetch(imageURL);
+              const blob = await res.buffer();
+
+              const uploadedImage = await s3
+                .upload({
+                  Bucket: AWS_S3_BUCKET_NAME,
+                  Key: `${AWS_S3_IMAGE_FOLDER}${imageKey}.jpg`,
+                  Body: blob,
+                })
+                .promise();
```

Comment:
To abstract into another arrow function as with the image deletion code.

Reply:
For now, I've abstracted the `Bucket` and `Key` properties into another function. The `Body` and `ContentType` properties would be hardcoded for the upload function and not be abstracted, since they are not used anywhere else, to prevent confusion